readme.md

```diff
@@ -85,6 +85,12 @@ In order to use the new photos email notification you will need to have configur
 
 Take a look at the [Documentation](https://lycheeorg.github.io/docs/), particularly the [FAQ](https://lycheeorg.github.io/docs/faq.html) if you have problems. Discovered a bug? Please create an issue [here](https://github.com/LycheeOrg/Lychee/issues) on GitHub! You can also contact us directly on [gitter (login with your github account). &#187;](https://gitter.im/LycheeOrg/Lobby)
 
+## Open Source Community Support
+
+<img src="https://resources.jetbrains.com/storage/products/company/brand/logos/PhpStorm_icon.png" alt="PhpStorm" width="50"/>
```

Comment:
This upsets me on several levels. Mostly downloading a 200kb png for a 50px logo, which looks fairly clunky anyway.

Reply:
Agreed. :( That's their logo though and they don't provide lower res (but do provide svg). :(